examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,130 @@
+
+X, y = make_classification(n_samples=700, random_state=rng)
+
+clf = RandomForestClassifier(n_estimators=20, random_state=rng)
+
+param_dist = {"max_depth": [3, None],
+              "max_features": randint(1, 11),
+              "min_samples_split": randint(2, 11),
+              "bootstrap": [True, False],
+              "criterion": ["gini", "entropy"]}
+
+rsh = HalvingRandomSearchCV(estimator=clf, param_distributions=param_dist,
+                            factor=2, random_state=rng)
+rsh.fit(X, y)
+rsh.best_params_
+
+##############################################################################
+# New SequentialFeatureSelector transformer
+# -----------------------------------------
+# A new iterative transformer to select features is available:
+# :class:`~sklearn.feature_selection.SequentialFeatureSelector`.
+# Sequential Feature Selection can add features once at a time (forward
+# selection) or remove features from the list of the available features
+# (backward selection), based on a cross-validated score maximization.
+# See the :ref:`User Guide <sequential_feature_selection>`.
+
+from sklearn.feature_selection import SequentialFeatureSelector
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.datasets import load_iris
+
+X, y = load_iris(return_X_y=True, as_frame=True)
+feature_names = X.columns
+knn = KNeighborsClassifier(n_neighbors=3)
+sfs = SequentialFeatureSelector(knn, n_features_to_select=2)
+sfs.fit(X, y)
+sfs_features  = [i for (i, v) in zip(feature_names, sfs.get_support()) if v]
+print("Features selected by forward sequential selection: "
+      f"{sfs_features}")
+
+##############################################################################
+# New PolynomialCountSketch kernel approximation function
+# -------------------------------------------------------
+
+##############################################################################
+# Individual Conditional Expectation
+# ----------------------------------
+
+##############################################################################
+# New metrics available
+# ---------------------
+
+##############################################################################
+# DecisionTreeRegressor now supports the new 'poisson' splitting criterion 
+# ------------------------------------------------------------------------
+
+##############################################################################
+# Retrieving datasets from literature as pandas dataframes
+# --------------------------------------------------------
+
+##############################################################################
+# HistGradientBoostingClassifier improved performances
+# ----------------------------------------------------
+
+##############################################################################
+# New documentation improvements
+# ------------------------------
+#
+# `New examples and documentation pages
+# <../../search.html?q=versionadded 0.24>`_
```

Comment:
I can't check what this page looks like (artifcats aren't rendered) but is this really needed? Why not just list the pages below, as it's already done?

(also there's a space here but not in the labels (`versionadded024`))

Reply:
> I can't check what this page looks like (artifcats aren't rendered)

Artifacts are rendered but for some reason they are not displayed in the artifact page: you can find them [here](https://316-178013228-gh.circle-artifacts.com/0/doc/auto_examples/release_highlights/plot_release_highlights_0_24_0.html).

> is this really needed? Why not just list the pages below, as it's already done?

Because I was willing to tag other examples and documentation pages, without linking them.
And also because the goal was to propose a implementation for #16601

> (also there's a space here but not in the labels (`versionadded024`))

Right, I will fix this.

Reply:
Thanks for the link

> And also because the goal was to propose a implementation for #16601

IMO This should be discussed in  #16601 then, because the scope of this PR is too restricted to properly appreciate the proposed solution for the example gallery.

To be on the safe side, it might be better to just list the examples manually here. In the search page (https://316-178013228-gh.circle-artifacts.com/0/doc/search.html?q=versionadded0.24), the list of examples doesn't look good (some inline html, and some irrelevant description)

Reply:
> IMO This should be discussed in #16601 then, because the scope of this PR is too restricted to properly appreciate the proposed solution for the example gallery.

Sorry if I don't feel out of scope, this isn't the first time that things are discussed in more than one place in the scikit-learn issue tracker. Anyway, my comment is written down, feel free to comment or not.

> To be on the safe side, it might be better to just list the examples manually here.

Knowing myself, the safe side is not copying manually text and web links ... :) Also as [you said](https://github.com/scikit-learn/scikit-learn/pull/18795#discussion_r520601705) the page is already quite long for a bullet list. I'm leaving things this way for now if you don't mind.

Reply:
> Sorry if I don't feel out of scope, this isn't the first time that things are discussed in more than one place in the scikit-learn issue tracker. 

It's not out of scope, but we cannot make an informed decision on the original issue by just looking at how things look in this PR.  #16601  involves more than just tagging examples.

I'm not saying that this isn't a promising solution for #16601, it might be. But we can't tell for now, and since this PR needs to be merged soon, it might be preferable to delay the  discussion for #16601 later.

> Anyway, my comment is written down, feel free to comment or not.

Sorry, I'm not sure which comment you are referring to?

> Knowing myself, the safe side is not copying manually text and web links 

By "safe side", I mean in terms of our decision process. Time is pressing, and we don't want to start implementing something here that isn't needed or that hasn't been proven to be needed yet. Also as I said, I personally don't find the search list to be user-friendly.

> Also as you said the page is already quite long for a bullet list

It's all relative, and it's about trade-offs. If the alternative to adding a little more text to a page is to have a [poorly rendered list](https://316-178013228-gh.circle-artifacts.com/0/doc/search.html?q=versionadded0.24), then it's still better to add the text.